CIP-1694/README.md

```diff
@@ -735,7 +737,6 @@ DRep registration certificates include:
 
 * a DRep ID
 * a deposit
-* a stake credential (for the deposit return)
 * an optional anchor
```

Comment:
This anchor is not mentioned in [New Metadata structures](https://github.com/cardano-foundation/CIPs/blob/ca632435b32cf5298d5200efe7a5a6754db3ced3/CIP-1694/README.md#new-metadata-structures) section, might be nice to justify this anchor to match the ones for governance actions and votes.

Reply:
Thank you, I'll add that in with whatever else people have found next week.